_posts/2024-03-22-kraft-migration.md

```diff
@@ -0,0 +1,392 @@
+---
+layout: post
+title:  "From ZooKeeper to KRaft: let's automate the migration with Strimzi"
```

Comment:
I wonder if the tile might be misread as that Strimzi can migrate non-Dtrimzi-managed clusters.

Reply:
Something like "Migrate your Strimzi-operated cluster from ZooKeeper to KRaft"?